README.md

```diff
@@ -105,9 +106,9 @@ Licenses
   and is licensed under the terms of the
   [LGPLv2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1).
 
-## Contributors
+## Financial Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](.github/CONTRIBUTING.md)].
+This project exists thanks to all the people who donate. [[Donate](.github/CONTRIBUTING.md)].
```

Comment:
`.github/CONTRIBUTING.md` is not just about financial contributions.
And IMHO should not.

`https://opencollective.com/kivy/contributors.svg` is just showing up all our Github (code) contributors.

Instead, below we have the backers one, dedicated to OpenCollective (financial) backers.

Reply:
Ahh. I see my misunderstanding. Sorry.

Done.

We have yet another duplicate of the same information. In the *other* review branch (about the FAQ and `docs/Contributing.rst`) I will update `.github/CONTRIBUTING.md` to also be in sync.